osu.Game/Collections/DrawableCollectionListItem.cs

```diff
@@ -163,19 +173,41 @@ private void load(OsuColour colours)
                     // but that subscription does not only cover *changes to the set of collections* (i.e. addition/removal/rearrangement of collections),
                     // but also covers *changes to the properties of collections*, which `BeatmapMD5Hashes` is one.
                     // when a collection item changes due to `BeatmapMD5Hashes` changing, the list item is deleted and re-inserted, thus guaranteeing this to work correctly.
-                    int count = collection.PerformRead(c => c.BeatmapMD5Hashes.Count);
+                    updateCountText();
 
-                    countText.Text = count == 1
-                        // Intentionally not localised until we have proper support for this (see https://github.com/ppy/osu-framework/pull/4918
-                        // but also in this case we want support for formatting a number within a string).
-                        ? $"{count:#,0} item"
-                        : $"{count:#,0} items";
+                    // subscribe to all BeatmapInfo changes
+                    subscription = realmAccess.RegisterForNotifications(
```

Comment:
So this, in total, is going to take out as many subscriptions as there are collections in this list? That's going to be a very difficult one to accept for me. People have _a lot_ of collections.